src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const resultInverse = {};
+  const seenValues = new Set();
+
+  for (const key in robot) {
+    if (robot.hasOwnProperty(key)) {
```

Comment:
it's unnecessary, you are literally iterating over object and it will have this key in 100% of the time